### PR TITLE
Add oVirt engine cleanup role

### PR DESCRIPTION
**Pull request by pkubica**

### CODE REVIEW

File: roles/ovirt-engine-cleanup/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Dependencies
+------------
+
+None
+
+Example Playbook
+----------------
+
+    hosts: engine
```
Comment (lukas-bednar) [inline roles/ovirt-engine-cleanup/README.md]: please use highlighting section for examples, like:

```yaml
---
hosts: engine
  remote_user: root
  vars:
    ....
    ....
```

File: roles/ovirt-engine-cleanup/templates/answerfile_4.1.txt.j2 [hunk 2]
```diff
@@ -0,0 +1 @@
+answerfile_4.0.txt.j2
```
Comment (StLuke) [inline roles/ovirt-engine-cleanup/templates/answerfile_4.1.txt.j2]: use naming 4.x and not symlinks, there should be no big changes in Y version releases

Comment (pkubica) [inline roles/ovirt-engine-cleanup/templates/answerfile_4.1.txt.j2]: OK, so we should modify this in other roles too

**Comment by lukas-bednar** [comment]
it deployed engine successfully on my environment

**Pull request merged**